docs/Anjay_integrations/ESP32_integration/M5Stick.md

```diff
@@ -13,7 +13,27 @@ Integrate your ESP32-based device to manage it via Coiote DM.
 0. Create a project directory for the integration.
 0. Go to [https://github.com/AVSystem/Anjay-esp32-client/releases](https://github.com/AVSystem/Anjay-esp32-client/releases) and download `m5stickc-plus.bin` and `nvs_partition_gen.py` to your project directory.
 
-## Step 2: Configure the client using an NVS partition
+## Step 2: Add device to Coiote DM
+
+To connect your M5StickC to the Coiote IoT Device Management LwM2M Server, use your access to a Coiote DM installation, or register at https://eu.iot.avsystem.cloud/ to get access.
+
+To connect the board:
+
+0. Log in to Coiote DM and from the left side menu, select **Device Inventory**.
+0. In **Device Inventory**, click **Add device**.
+0. Select the **Connect your LwM2M device directly via the Management server** tile.
+![Add via Mgmt](images/mgmt_tile.png "Add via Mgmt")
+0. In the **Device credentials** step:
+    - In the **Device ID** field, type the endpoint name provided in the `nvs_config.csv`, e.g. `ESP32_test`.
+    - In the **Security mode** section, select the **PSK** mode.
+    - In the **Key identity** field, type the name provided in the `nvs_config.csv`, e.g. `ESP32_test`.
+    - In the **Key identity** field, type the identity provided in the `nvs_config.csv`, e.g. `ESP32_test`.
```

Comment:
Doubled the `Key identity` step. Also it is worth mentioning that `It is recommended to use endpoint name as key identity.`